modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -0,0 +1,195 @@
+= Deploy Applications to On-Premises Mule Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can deploy Mule Applications to your On-Premises Mule Instances using any of the following tools:
+
+* Anypoint Runtime Manager +
+See xref:runtime-manager::deploying-to-your-own-servers.adoc[Deploying to Your Own Servers] for instructions.
+* Anypoint CLI +
+See the xref:runtime-manager::anypoint-platform-cli-commands.adoc#standalone-management[runtime-mgr standalone-application deploy] command documentation for instructions.
+* Anypoint Studio +
+See xref:studio::import-export-packages.adoc#export-project-studio[Exporting Projects From Studio] for instructions. 
+* Mule Maven Plugin +
+Follow the instructions in this article to deploy to Runtime Fabric using Mule Maven Plugin.
```

Comment:
Ditto edits to PR #1014 and 1015, same text.

Reply:
Actually, I am ending the edit here. This is all the same text as in the previous two PRs. 
Please apply the edits made in those two PRs to all copies of the same text among the three topics.